### PR TITLE
Add discord link to navbar

### PR DESCRIPTION
**Pull request by sethdavis512**
| Mobile | Desktop |
|---|---|
| add-discord-link-mobile | add-discord-link-desktop |

### CODE REVIEW

File: app/components/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -2,11 +2,14 @@ import { Link } from "@remix-run/react";
 import GitHubIcon from "../icons/GitHubIcon";
 import MeetupIcon from "../icons/MeetupIcon";
 import YouTubeIcon from "../icons/YouTubeIcon";
+import DiscordIcon from "../icons/DiscordIcon";
 
 export const homepageLinkTitle = "Remix Austin 💿";
 export const meetupUrl = "https://www.meetup.com/remix-austin/";
 export const gitHubUrl = "https://github.com/remix-austin";
 export const youTubeUrl = "https://www.youtube.com/@remixaustin";
+export const discordUrl =
```
Comment (grantnorwood) [inline app/components/Navbar/Navbar.tsx:11]: Should this be the invite link?  If we want it to come from Brooks with his name, does he need to create a unique invite url?  Should we keep using this channel url, but explain how to get invited to the Remix server in the card below or something?

cc: @brookslybrand

Comment (sethdavis512) [reply app/components/Navbar/Navbar.tsx:11]: > If we want it to come from Brooks with his name, does he need to create a unique invite url?
That's my understanding, yes.

> Should we keep using this channel url, but explain how to get invited to the Remix server in the card below or something?
I'm not super knowledgeable on all things Discord. I wanna say the invite link is the right way to go and Discord will do the explaining 🤷🏻

Comment (brookslybrand) [reply app/components/Navbar/Navbar.tsx:11]: So sorry, y'all have been moving way faster than I've been able to (which is awesome!)

Try this link? https://discord.gg/mYy2kzqX

**Comment by sethdavis512** [comment]
I messed this MR up...closing in favor of #42

**Pull request closed without merging**